deploy/crds/toolchain.openshift.dev_tektoninstallations_crd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: tektoninstallations.toolchain.openshift.dev
+spec:
+  group: toolchain.openshift.dev
+  names:
+    kind: TektonInstallation
+    listKind: TektonInstallationList
+    plural: tektoninstallations
+    singular: tektoninstallation
+  scope: Namespaced
```

Comment:
And btw, I think we should make it cluster scoped. And the same for `CheInstallation`.

Reply:
Addressed 942585f